build/commands/lib/util.js

```diff
@@ -68,12 +68,22 @@ async function applyPatches() {
   Log.progressFinish('apply patches')
 }
 
+// Always rebuild typedef export to avoid linkage error.
+const isOverrideExportTypedef = (override) => {
+  const exportedHeaders = ['print_view_manager.h'];
+  if (exportedHeaders.includes(path.basename(override))) {
+    return true
+  }
+  return false
+}
```

Comment:
this doesn't look right. What exactly are you trying to solve here? What the errors are you seeing?

Reply:
hmm, I can't reproduce the errors now. It was complaining about undefined reference to `PrintViewManager::FromWebContents` and also `PrintViewManager::UserDataKey` IIRC.
But with https://github.com/brave/brave-core/pull/22612#discussion_r1527881207, this is definitely not needed

Reply:
removed in 2f09315c092a174bd6468e7f277e5ced7a5e5036